### PR TITLE
Refactor data import

### PR DESCRIPTION
**Pull request by seocam**
Needs:
* [ ] Update docs
* [x] Update readme

### CODE REVIEW

**Comment by seocam** [comment]
Over to @GustJc for review

File: vagrant/centos.sh [hunk 1]
```diff
@@ -22,9 +22,9 @@ yum install -y epel-release
 
 yum -y groupinstall "Development tools"
 
-yum install -y git unzip gettext libxml2-devel libxslt-devel openssl-devel libffi-devel python-devel python-pip python-virtualenvwrapper rabbitmq-server
+yum install -y git unzip gettext libxml2-devel libxslt-devel openssl-devel libffi-devel python-devel python-pip python-virtualenvwrapper redis-server
```
Comment (GustJc) [inline vagrant/centos.sh]: There is no redis-server in yum install.... Please refer to https://gist.github.com/nghuuphuoc/7801123 for a installation of redis

File: README.rst [hunk 2]
```diff
@@ -64,7 +64,7 @@ To run Colab with development server you will have to:
 
 .. code-block::
 
-  colab-init-config > /etc/colab/settings.yaml
+  colab-admin initconfig > /etc/colab/settings.yaml
```
Comment (GustJc) [inline README.rst]: settings.yaml does not exist anymore, should be settings.py

File: colab/plugins/tasks.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+from datetime import timedelta
+from celery.decorators import periodic_task
+
+from .data import TASKS
+
+
+@periodic_task(run_every=timedelta(seconds=5))
```
Comment (GustJc) [inline colab/plugins/tasks.py]: Just a remainder, this should be set to at least 1 minute.

**Pull request merged**